pkg/kubelet/kubelet.go

```diff
@@ -1597,9 +1597,13 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 			}
 		}
 		if mirrorPod == nil || deleted {
-			glog.V(3).Infof("Creating a mirror pod for static pod %q", format.Pod(pod))
-			if err := kl.podManager.CreateMirrorPod(pod); err != nil {
-				glog.Errorf("Failed creating a mirror pod for %q: %v", format.Pod(pod), err)
+			if _, err := kl.GetNode(); err != nil {
```

Comment:
Do we need to check the deletion timestamp?

Reply:
Yes, we should support the graceful deletion. Will add it. Thanks for pointing out.